app/components/containers/containersController.js

```diff
@@ -128,7 +130,11 @@ angular.module('containers', [])
           });
         }
         else if (action === Container.remove) {
-          action({id: c.Id}, function (d) {
+          var force = false;
+          if (c.State === 'running') {
```

Comment:
In the future, if we have another item calling the remove, function, we have to take care to call confirmRemoveAction, and not removeAction directly
But yeah, in the containers view, i use force=true in all cases, so we can do that here too